kuzzleio/kuzzle-admin-console
# Add import/export API actions

## What does this PR do ?
Add import/export features to API Actions view

Buttons import/export to left corner in sidebar
export:
  - open modal with table listing all the saved queries of all env (in the localStorage)
  - allow to check one or more queries
  - export button modal-ok

import: 
  - open modal with file input (file explorer/drag n drop)
  - multifile import supported
  - list of files to import (size and remove button)
  - list of queries to import 
  - allow to edit queries name before import (inputstate (red/green) = name already used)

Tests added:
  - Export a query in file and check file content
  - Import a query with file 
  - Change a query name while importing

### How should this be manually tested?
  - Step 1 : create and save some queries
  - Step 2 : click the export button and check the file
  - Step 3 : delete your previous saved queries
  - Step 4 : click the import button and upload the previous downloaded file

### Other changes
/

### Boyscout
/

![Screenshot from 2021-07-01 17-50-55](https://user-images.githubusercontent.com/44427849/124155211-89655080-da96-11eb-808b-1a96eeed7e74.png)

![Screenshot from 2021-07-01 17-59-37](https://user-images.githubusercontent.com/44427849/124155253-96823f80-da96-11eb-9da2-61f6cfa84950.png)

![Screenshot from 2021-07-01 17-59-17](https://user-images.githubusercontent.com/44427849/124155276-9d10b700-da96-11eb-8ed1-430a9e0d7318.png)
